webapp/src/components/user_attribute/user_attribute.tsx

```diff
@@ -1,17 +1,9 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
 
-export default class UserAttribute extends React.PureComponent {
-    static propTypes = {
-        id: PropTypes.string.isRequired,
-        username: PropTypes.string,
-        gitlabURL: PropTypes.string,
-        actions: PropTypes.shape({
-            getGitlabUser: PropTypes.func.isRequired,
-        }).isRequired,
-    };
+import {UserAttributeProps} from '.';
```

Comment:
To avoid a circular import

```suggestion
import type {UserAttributeProps} from '.';
```